hub/api/dataset.py

```diff
@@ -130,9 +131,10 @@ def __getitem__(
     def create_tensor(
         self,
         name: str,
-        htype: str = DEFAULT_HTYPE,
-        chunk_size: int = None,
-        dtype: str = None,
+        htype: Optional[str] = DEFAULT_HTYPE,
+        chunk_size: Optional[int] = None,
+        dtype: Optional[str] = None,
```

Comment:
you can remove `Optional` from here.

mypy won't give an error if `dtype: str = None` (this is implicit `Optional`)